vendor/k8s.io/kubernetes/pkg/util/iptables/iptables.go

```diff
@@ -134,20 +128,19 @@ const LockfilePath16x = "/run/xtables.lock"
 type runner struct {
 	mu              sync.Mutex
 	exec            utilexec.Interface
-	dbus            utildbus.Interface
 	protocol        Protocol
 	hasCheck        bool
+	hasListener     bool
```

Comment:
Minor backporting error in "UPSTREAM 81517: Add iptables.Monitor, use it from kubelet and kube-proxy": this field shouldn't have gotten added (and likewise the initialization of it to `false` below in `newInternal`). (The field previously existed in 4.2, but not 3.11, but it got removed in 4.2 as part of this commit.)

Reply:
done